data_steward/validation/main.py

```diff
@@ -882,53 +881,47 @@ def _is_string_excluded_file(gcs_file_name):
 
 
 def process_hpo_copy(hpo_id):
-    """copies over files from hpo bucket to drc bucket
-
+    """
+    copies over files from hpo bucket to drc bucket
     :hpo_id: hpo from which to copy
     """
+
+    storage_client = StorageClient()
+    bucket_items: list = []
+
     try:
-        storage_client = StorageClient()
         hpo_bucket = storage_client.get_hpo_bucket(hpo_id)
-        drc_private_bucket = storage_client.get_drc_bucket()
-        source_bucket = storage_client.get_bucket(hpo_bucket)
-        destination_bucket = storage_client.get_bucket(drc_private_bucket)
-        bucket_items = list_bucket(hpo_bucket)
-
-        ignored_items = 0
-        filtered_bucket_items = []
-        for item in bucket_items:
-            item_root = item['name'].split('/')[0] + '/'
-            if item_root.lower() in common.IGNORE_DIRECTORIES:
-                ignored_items += 1
-            else:
-                filtered_bucket_items.append(item)
+        drc_bucket = storage_client.get_drc_bucket()
+        bucket_items: list = storage_client.get_bucket_items_metadata(
+            hpo_bucket)
+    except BucketNotSet:
+        logging.info(f"Bucket for hpo_id '{hpo_id}' is empty/unset")
```

Comment:
nit: can use the passed message
```suggestion
    except BucketNotSet as e:
        logging.info(e.message)
```